src/main/java/core/basesyntax/service/CsvFileWriterService.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.service;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class CsvFileWriterService implements WriterService {
+    @Override
+    public void writeToFile(final String data, final String filePath) {
+        File file = new File(filePath);
+        try (
+                FileWriter fileWriter = new FileWriter(file);
+                BufferedWriter bufferedWriter = new BufferedWriter(fileWriter)
+        ) {
+            bufferedWriter.write("");
+            bufferedWriter.write(data);
+        } catch (IOException e) {
+            throw new RuntimeException("File " + e + " not found.");
+        }
```

Comment:
Remember the Principle Solid. Here, this service should only work by writing to the file what we pass to it.